tests/mock_vllm_model.py

```diff
@@ -286,24 +272,24 @@ def decode_forward(
         # logits: [batch, seqlen, vocab_size]
         logits = torch.randn((batch, seqlen, 128256))
         # send a token every period loops
-        EOT_ID = 128009
-        # EOS_ID = 128001
-        send_index = 200
-        send_token = EOT_ID
-        if start_pos is not None:
-            if isinstance(start_pos, int):
-                # if start pos is same across batch, ie. now in prefill
-                cache_idxs = torch.tensor(
-                    [start_pos for _ in range(batch)], dtype=torch.int64
-                )
-            else:  # if start_pos is a tensor ie. is different across batch, now in decode mode
-                # if start position is greater than index to send EOT
-                cache_idxs = start_pos.to(dtype=torch.int64)
-                send_token_mask = cache_idxs > send_index
-                # find positions where start pos passes send_index (ie. done decoding) + make 1D
-                batch_indices = torch.nonzero(send_token_mask).squeeze()
-                # assign a high logit at at the send _token index so model will select it and generate the EOT so that generation stops
-                logits[batch_indices, 0, send_token] = 100.0
+        # EOT_ID = 128009
```

Comment:
Should we leave this commented out?

Reply:
I'm leaving the code block in case we need it to test a specific case where a token gets sent at specific time/output length. If thats unattractive I can break it out into a function that is turned off.

Reply:
added in https://github.com/tenstorrent/tt-inference-server/pull/43/commits/1e82b4201e0300560181067ccc1b389f07a96c40